app/pods/application/route.js

```diff
@@ -75,5 +76,8 @@ export default Route.extend(ApplicationRouteMixin, {
       later(function(){
         controller.set('code', undefined)
       })
+    },
+    afterModel(model) {
+      this.set('headData.title', 'Coding Blocks Online')
```

Comment:
Remove space between Coding and Blocks.

Reply:
Let the space be there, we can target three different key words,
1. Coding
2. Blocks
3. Coding Blocks